src/validation.cpp

```diff
@@ -916,6 +923,7 @@ bool MemPoolAccept::PreChecks(ATMPArgs& args, Workspace& ws)
     // blocks
     if (!bypass_limits && !CheckFeeRate(ws.m_vsize, ws.m_modified_fees, state)) return false;
 
+
```

Comment:
Nah, don't add a line here

Reply:
Removed in: https://github.com/SequentiaSEQ/SEQ-Core-Elements/pull/10/commits/f1473e638045ecc24a70b02f0afbd230c3a845f6